suricata.yaml.in

```diff
@@ -1091,6 +1091,19 @@ asn1-max-frames: 256
 #  user: suri
 #  group: suri
 
+# Use landlock security module under Linux
+landlock:
+  enabled: no
+  directories:
+    #write:
+    #  - @e_rundir@
+    # /usr and /etc folders are added to read list to allow
+    # file magic to be used.
+    read:
+      - /usr/
+      - /etc/
+      - @e_sysconfdir@
```

Comment:
I wonder if this should be under a `security` top-level field?  We don't have it yet, but see: https://github.com/OISF/suricata/pull/7458/files#diff-6edad567c7414ca3b78009e7d315547542cbb77455e7a223c11719d2734fd3db

Reply:
> I wonder if this should be under a `security` top-level field? We don't have it yet, but see: https://github.com/OISF/suricata/pull/7458/files#diff-6edad567c7414ca3b78009e7d315547542cbb77455e7a223c11719d2734fd3db

Makes sense to me.

Reply:
> > I wonder if this should be under a `security` top-level field? We don't have it yet, but see: https://github.com/OISF/suricata/pull/7458/files#diff-6edad567c7414ca3b78009e7d315547542cbb77455e7a223c11719d2734fd3db
> 
> Makes sense to me.

Do I introduce it in updated PR with adding the group/user change in the same category (may be complex on backward compatibility) ?

Reply:
Not sure, lets not block this work on that.

If we move `run-as.user` we should also consider the other "run" options. Probably better to do that discussion separately.

Reply:
Agreed.  Both landlock and process limits are new, so easy to consolidate into the same object now.